debian/rules

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --with systemd
+
+override_dh_auto_configure:
+	# Build the configure script if it doesn't already exist
+	-[ ! -x $(CURDIR)/configure ] && $(CURDIR)/autogen.sh
+	dh_auto_configure -- --with-math --with-webdir=/var/lib/netdata
+
+override_dh_install:
+	dh_install
+	# Documentation and JavaScript should not be executable
+	chmod 644 $(CURDIR)/debian/netdata/usr/lib/*/netdata/*/*.js
+	chmod 644 $(CURDIR)/debian/netdata/usr/lib/*/netdata/*/*.md
+	# This config file is also a script
+	chmod 755 $(CURDIR)/debian/netdata/etc/netdata/charts.d.conf
```

Comment:
should this be run using '.' (source)? if not, we should modify Makefile.am to dist_config_SCRIPTS and remove this.